README.md

```diff
@@ -130,7 +134,37 @@ This is not an issue when relying on `opam` to build the Coq project.
 Otherwise, you may want to use `sudo` in the container to change the
 permissions. You may also install additional Debian packages.
 
-For more details, see the
+Typically, this would lead to a workflow specification like this:
+
+```yaml
+runs-on: ubuntu-latest
+strategy:
+  matrix:
+    image:
+      - 'coqorg/coq:dev'
+steps:
+  - uses: actions/checkout@v2
+  - uses: coq-community/docker-coq-action@v1
+    with:
+      opam_file: 'coq-bignums.opam'
+      custom_image: ${{ matrix.image }}
+      custom_script: |
+        ...
+        startGroup Workaround permission issue
+          sudo chown -R coq:coq .  # <--
+        endGroup
+        startGroup Run tests
+          make -C tests
+        endGroup
+  - name: Revert permissions
+    # to avoid a warning at cleanup time
+    if: ${{ always() }}
```

Comment:
Is this line useful?

Reply:
I guess so: it should ensure that the final `chmod` is run even if the previous (docker-coq-action) step failed

Reply:
Sounds good.